api/config/base/rbac/role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: cf-admin-clusterrole
 subjects:
 - kind: ServiceAccount
-  name: cf-admin-serviceaccount
-  namespace: default
+  name: cf-k8s-api-cf-admin-serviceaccount
+  namespace: cf-k8s-api-system
```

Comment:
I think maybe this is a POC change, and we didn't mean to commit it?

Reply:
The [service-account](https://github.com/cloudfoundry/cf-k8s-controllers/blob/issues/380-delete-org/api/config/base/rbac/service_account.yaml) gets created in `cf-k8s-api-system` and get prefixed with `cf-k8s-api-`. I deployed this on Kind to verify the same. I think the changes are needed. If not for the changes, the ClusterRoleBinding will be pointing at a non existent service-account.

Reply:
@julian-hj You were correct. This has been addressed in commit 7ebed6d1bd42cdf09dec7b90d7e1cad28adfc590